src/loaders/utils/GetYoutubeTakeoverDetails.ts

```diff
@@ -0,0 +1,101 @@
+import { isServerSide } from 'utils/getEnv';
+import getValueFromQuery from 'utils/getValuefromQuery';
+import { getArticleVideoServiceBaseUrl } from 'utils/getPandoraDetails';
+import { trackYoutubeTakeoverDetails, YoutubePlayerTrackingProps } from 'youtube/players/shared/youtubeTrackingEvents';
+
+function getYoutubeTakeoverUrl(wikiId?: string): string {
+	const articleVideoBaseUrl = getArticleVideoServiceBaseUrl();
+	if (articleVideoBaseUrl.length === 0) {
+		return articleVideoBaseUrl;
+	}
+	if (!wikiId) {
+		return '';
+	}
+	return `${articleVideoBaseUrl}youtube/v1/youtube-takeover-mappings/${wikiId}`;
+}
+
+interface MWConfig {
```

Comment:
Should these be moved into `src/youtube/types.d.ts` ?

Reply:
Fixed in 954b9d45657bcecabad3eb63bf4152415f424650